ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/tile/tile.html

```diff
@@ -0,0 +1,10 @@
+
+tile component
+<div class='product-listing-tile' data-tag=${properties.tag}>
+<br>
+<div data-sly-resource="${'img' @ resourceType='hclecomm/components/content/image', decoration=true}"> </div>
+<br>
+<p style="text-align:center;">${properties.brand} </p>
```

Comment:
@ReenaOmana : why doe we have inline style here ?

Reply:
This is removed